src/pages/AddCar/AddCar.jsx

```diff
@@ -0,0 +1,120 @@
+import { FaPlus, FaTrash } from 'react-icons/fa';
+import React, { useEffect, useState } from 'react';
+import './index.css';
+
+const AddCar = () => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [color, setColor] = useState('');
+  const [year, setYear] = useState('');
+  const [price, setPrice] = useState('');
+  const [type, setType] = useState('');
+  const [formInfo, setFormInfo] = useState({});
+  const [images, setImages] = useState([]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setFormInfo(
+      {
+        ...formInfo,
+        name,
+        description,
+        color,
+        year,
+        price,
+        type,
+        images,
+      },
+    );
+    setName('');
+    setDescription('');
+    setColor('');
+    setYear('');
+    setPrice('');
+    setType('');
+    setImages([]);
+  };
+
+  useEffect(() => {
+    console.log(formInfo);
+  }, [formInfo]);
+
+  const addImage = () => {
+    const newImages = [...images, ''];
+    setImages(newImages);
+  };
+
+  const deleteImage = (i) => {
+    const newImages = [...images];
+    newImages.splice(i, 1);
+    setImages(newImages);
+  };
+
+  const handleImage = (e, index) => {
+    const newImages = [...images];
+    newImages[index] = e.target.value;
+    setImages(newImages);
+  };
+
+  return (
+    <section id="add_car">
+      <div className="form__container">
+        <h2>Add A Car</h2>
+        <form onSubmit={handleSubmit}>
+          <label htmlFor="name">
+            Name:
+            <input type="text" required onChange={(e) => setName(e.target.value)} id="name" value={name} />
+          </label>
+          <label htmlFor="description">
+            Description:
+            <textarea onChange={(e) => setDescription(e.target.value)} value={description} id="description" />
```

Comment:
I think it would be great if you can add validation of a max number of characters in the textarea too.